tests/characterization_data/filters_expected/sample_nsxv.nsx

```diff
@@ -1,54 +1,612 @@
-<!--
- $Id:$
- $Date:$
- $Revision:$
--->
+<?xml version="1.0" ?>
```

Comment:
Looks like it is fine to have, if it is not there it gets added when you extract it.